### ISSUE TITLE
Focus order is not in the sequence manner for the 'Comp terms' document on the page.

### ISSUE THREAD

**Issue by msftedad** [gdh1995/pdf.js#21]
Attach (recommended) or Link to PDF file here:
 https://mozilla.github.io/pdf.js/web/viewer.html.
[edad (2).pdf](https://github.com/gdh1995/pdf.js/files/14600757/edad.2.pdf)

Configuration:
- Operating system and its version: iPhone 13 Version-17.4
- PDF.js version: 1.4   

Pre-requisites:
Open Settings> Accessibility> Voiceover> Turn on Voiceover.

Steps to reproduce the problem:

1. Open this PDF viewer https://mozilla.github.io/pdf.js/web/viewer.html.
2. Navigate to 'Tools' option at header at header of pdf viewer.
3. Activate 'Open' control and upload any pdf (Having links in it) from your system.
4. Now you'll have the pdf file, in pdf viewer.
5. Navigate through PDF.
6. Observe the issue.

What is the expected behavior? (add screenshot)
Focus order should be in the sequence manner for the 'Comp terms' document on the page.

Navigating the document, focus must move in a logical sequence as using the right swipe gesture the whole title should have single focus and read it as one and focus should be in Top to Bottom and left to right sequence using right swipe in the document.

What went wrong? (add screenshot)
Focus order is not in the sequence manner for the 'Comp terms' document on the page.When navigating the document, focus does not move in a logical sequence as using the right swipe gesture the title is having separate focus and focus is moving in top and then in next swipe bottom then again focus is moving from the start in the document.

https://github.com/gdh1995/pdf.js/assets/93735775/50209110-4739-4f9c-989e-0a8248b7aa34

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by msftedad** [comment]
Hi @gdh1995, Any update on this issue? Please provide the ETA to fix this issue.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
